### PR TITLE
Test VM resource quotas

### PR DESCRIPTION
**Pull request by rajivnathan**
Tests for https://github.com/codeready-toolchain/host-operator/pull/1002

### CODE REVIEW

File: testsupport/tiers/checks.go [hunk 1]
```diff
@@ -529,7 +529,7 @@ type appstudiolargeTierChecks struct {
 
 func (a *appstudiolargeTierChecks) GetClusterObjectChecks() []clusterObjectsCheck {
 	return clusterObjectsChecks(
-		clusterResourceQuotaDeploymentCount("600", "100"),
+		clusterResourceQuotaDeploymentCount("600", "100", "2"),
```
Comment (alexeykazakov) [inline testsupport/tiers/checks.go:532]: It should be added to the base1ns tier check and all tiers which base on that tier. Not the appstudiolarge tier check. Right?

Comment (rajivnathan) [reply testsupport/tiers/checks.go:532]: ah, missed that this was an appstudio tier.

**Review by alexeykazakov — APPROVED** [review]
Looks good now!

**Review by ranakan19 — APPROVED** [review]
Sweet

**Comment by rsoaresd** [comment]
Hellooo,

The last run failed with this error (the same happens in the paired PR):

```
=== NAME  TestCreateVirtualMachine/vm-domain-memory-guest
    vm_webhook_test.go:88: 
        	Error Trace:	/go/src/github.com/codeready-toolchain/toolchain-e2e/test/e2e/parallel/vm_webhook_test.go:88
        	Error:      	Received unexpected error:
        	            	virtualmachines.kubevirt.io "domain-memory-guest" is forbidden: exceeded quota: for-test-vm-deployments, requested: count/virtualmachines.kubevirt.io=1, used: count/virtualmachines.kubevirt.io=2, limited: count/virtualmachines.kubevirt.io=2
        	Test:       	TestCreateVirtualMachine/vm-domain-memory-guest
```

**Pull request merged**